newsroom/cards/module.py

```diff
@@ -26,11 +27,16 @@
 async def get_settings_data():
     app = get_current_wsgi_app()
 
+    cards_task = CardsResourceService().find({})
+    navs_task = NavigationsService().search(lookup={"is_enabled": True})
+
+    cards, navigations = await gather((await cards_task).to_list_raw(), (await navs_task).to_list_raw())
```

Comment:
Using `await` is a synchronous action, meaning this will await `cards_task` first, then await `navs_task` sequentially. To gain the benefit of concurrently awaiting both requests for a cursor, we will have to use something like:
 ```python
# Await the request for a cursor
[cards_cursor, navs_cursor] = await gather(cards_task, navs_task)
# Await for the cursor data
[cards, navigations] = await gather(cards_cursor.to_list_raw(), navs_cursor.to_list_raw())
```

This is because the initial request gets sent to MongoDB, which returns a cursor to the request. This is an async I/O request.

Then `to_list_raw` then requests MongoDB to return items from the cursor. This is another async I/O request.

So because we have 2 separate I/O requests to MongoDB, we will need to gather twice.

Reply:
Sorry, I got my information wrong here. Your changes here should suffice for mongo resources.

**Mongo:**
mongo.find: sync
cursor: async

**Elastic**
elastic.search: async
cursor: sync